groups/chips-uam-infrastructure/alb.tf

```diff
@@ -10,7 +10,7 @@ module "chips_uam_internal_alb_security_group" {
   vpc_id      = data.aws_vpc.vpc.id
 
   ingress_cidr_blocks = local.admin_cidrs
-  ingress_rules       = ["http-80-tcp", "https-443-tcp"]
+  ingress_rules       = ["http-80-tcp", "https-443-tcp", "http-8080-tcp"]
```

Comment:
Shouldnt need 8080 rule here, users should access it it on 80/443 and the ALV translates that to 8080 on backend.

Reply:
Updated to   ingress_rules       = ["http-80-tcp", "https-443-tcp"]